src/test/java/edu/byu/cs/util/RepoUrlValidatorTest.java

```diff
@@ -121,7 +121,7 @@ void should_throwException_when_givenMalformedOrNotRelatedUrl() {
         };
         for (String badUrl : badUrls) {
             assertThrows(RepoUrlValidator.InvalidRepoUrlException.class,
-                    () -> RepoUrlValidator.clean(badUrl),
+                    () -> RepoUrlValidator.cleanRepoUrl(badUrl),
```

Comment:
Generally, we want to make changes that minimize the required changes to external files. The code had previously been organized in a way that would minimize these dependencies. This is a key example of other negative effects of removing the `clean()`, besides the comments already made.